packages/mostly-equal/src/test/mostly-equal-matchers.unit.ts

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { expect } from 'chai';
+import { notImportant, equal, defined, defineUnique, defineSame, thumbsUp } from '../index';
+
+describe('mostly equal matchers', () => {
+  describe('notImportant', () => {
+    it('should not throw for fields marked with not important', () => {
+      expect(() => {
+        expect({
+          a: 1,
+        }).to.mostlyEqual({
+          a: notImportant,
+          b: notImportant,
+        });
+      }).not.to.throw();
```

Comment:
nitpick: `.to.not.throw()` is a bit more readable in my eyes. kinda like reading `!throw()` :)

Reply:
done